contracts/liquidity-protection/LiquidityProtection.sol

```diff
@@ -83,6 +84,12 @@ contract LiquidityProtection is ILiquidityProtection, Utils, Owned, ReentrancyGu
         uint128 removeAverageRateD; // average rate of 1 A in units of B when liquidity is removed (denominator)
     }
 
+    struct MigrationUnit {
```

Comment:
`AggregatedPositions`?

Reply:
That won't work, because the main function takes `MigrationUnit[] migrationUnits` as input.

Reply:
not following you, it's a rename.
`AggregatedPositions aggregatedPositions`

Reply:
You're suggestion a rename from single to plural, which means that we'll need to use:
```
AggregatedPositions[] aggregatedPositionss
```
Which sounds pretty bad.

Reply:
we can just use -
AggregatedPositions aggregatedPositions
AggregatedPositions[] aggregatedPositions
it's not perfect but it's clear enough and much more self explanatory than MigrationUnit which is very generic and doesn't help the user understand anything about the content

Reply:
Renamed to `PositionList`.
LMK if you want to add or change to `AggregatedPositions` or `AggregatedPositionList` or w/e.

Reply:
Clear enough.